client/package.json

```diff
@@ -138,6 +148,7 @@
     "react-confirm-bootstrap": "5.3.1",
     "react-container-dimensions": "1.4.1",
     "react-dom": "16.13.1",
+    "react-moment-proptypes": "^1.7.0",
```

Comment:
Pin version number (`yarn add -E react-moment-proptypes`) as that is what _renovate_ likes.